### DISCUSSION TITLE
[Question] How to get actual file paths with filePicker on mobile

### DISCUSSION THREAD

**Discussion by Dzheremi2** [flet-dev/flet#3990] [Q&A]
### Question

File picker returns file path like `/data/user/0/com.appveyor.flet/cache/file_picker`.
Is there any method to get actual file path like `/storage/emulated/0/`

### -------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 0Liposo0** [comment]
Are you using PWA? If so, both IOS and Android do not return the real paths, if you create an executable, and on Android for example, it will return the real path

**Reply by Dzheremi2** [reply]
I am using APK generated by flet build command.
If i clone my app's repo on my phone and run it with `flet run --android` command without `sudo` the flet app will return error, that it doesn't have access to its own files. 
`Permission denied: '/data/user/0/com.appveyor.flet/cache/file_picker/1730647561859/BANNERS - Someone To You.mp3'`

Because of i can't see builded app logs, i don't know if the error is the same but still.
